inventory/version_test.go

```diff
@@ -0,0 +1,39 @@
+package inventory
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestVersion(t *testing.T) {
+	type VersionResponse struct {
+		Version string
+	}
+
+	t.Run("Get", func(t *testing.T) {
+		url := fmt.Sprintf("%sv1/version", *pmmapitests.ServerURLF)
```

Comment:
Please test 3 endpoints:
* `/ping`
* `/managed/v1/version`
* `/v1/version`
All of them are important.

Also do not construct URL that way. `main()` has special handling of URLs with missing `/` in the end, etc. Probably we should store URL in s package-level variable and use there instead of raw flag value.